README.md

```diff
@@ -46,6 +46,7 @@ can be found under [documentation][examples].
 defmodule Consumer do
   @behaviour GenRMQ.Consumer
 
+  @impl GenRMQ.Consumer
```

Comment:
This looks like an annotation of sort. What does it do?
Is this change strictly related to the purpose of the PR, or just a change you where doing "while you were at it"?

Reply:
It is Elixir way of documenting which functions are callbacks

Reply:
Got it, thanks.